charts/misc-apps/templates/signalilo.yaml

```diff
@@ -0,0 +1,33 @@
+{{ if .Values.signalilo.enabled }}
+{{ template "argoconfig.application" (list . "misc-apps.signalilo") }}
+{{ end }}
+
+{{- define "misc-apps.signalilo" -}}{{- $app := unset .Values.signalilo "enabled" -}}{{- $name := default $app.namespace $app.name -}}
```

Comment:
The same point as pointed out for the analogue line in downscaler.yaml. Simply replace "downscaler" with "signalilo".

Reply:
also in 3ece9eb1ccf527e93e7ded1ab7e5ca00e2880f3f